CHANGELOG.md

```diff
@@ -70,6 +70,10 @@
   [keith](https://github.com/keith)
   [#4782](https://github.com/realm/SwiftLint/issues/4782)
 
+* Move `excludedPaths` out of iteration loop in favor of dependency injection  
```

Comment:
This could be more informative in that it describes what the change means for users. The current entry is quite technical and an internal detail.

Also, please finish the entry with a period and two spaces to enforce a line break in Markdown.

Sorry missed that in the last review ...

Reply:
Sure np. I've changed it to:
> Move `excludedPaths(fileManager:)` operation out of linting iterations
  in favor of dependency injection to improve build speeds for Swift Plugin.